terraform/helm/fullnode/values.yaml

```diff
@@ -12,16 +12,6 @@ fullnode_identity:
   # Set this to start fullnode from an existing key.
   # If not set, it will generate a random one on startup.
 
-fullnode_inbound_rate_limit:
-  enabled: true
```

Comment:
Do you know if the default for rate limiting is enable or not? we can set this to false to have it disabled by default, or else if the code has enabled, it will still use the rate limit number in the default value

Reply:
As far as I can see, the default is disabled 😄 https://github.com/aptos-labs/aptos-core/blob/65ed6c0e469bc0512806cefbd8c7947431892276/config/src/config/network_config.rs#L133